### PR TITLE
webrtc: convert RTCSignalingChannel to FFmpegInput + consolidate all converters

### PR DESCRIPTION
**Pull request by bjia56**
I was able to test this by converting a WebRTC mixin device (since that implements RTCSignalingChannel) to FFmpegInput, then ffprobe-ing the returned RTSP.

### CODE REVIEW

**Comment by koush** [comment]
the multiple buffer converters thing is getting unwieldy, this was added a while ago

image

**Comment by bjia56** [comment]
That looks perfect, I can see about porting the old RTC bridge over

File: plugins/webrtc/src/main.ts [hunk 1]
```diff
@@ -772,4 +784,41 @@ class WebRTCBridge extends ScryptedDeviceBase implements BufferConverter {
...
+class WebRTCFFmpegBridge extends ScryptedDeviceBase implements BufferConverter {
+    constructor(public plugin: WebRTCPlugin, nativeId: string) {
+        super(nativeId);
+
+        this.fromMimeType = ScryptedMimeTypes.RTCSignalingChannel;
+        this.toMimeType = ScryptedMimeTypes.FFmpegInput;
+    }
+
+    async convert(data: any, fromMimeType: string, toMimeType: string, options?: MediaObjectOptions): Promise<any> {
+        const channel = data as RTCSignalingChannel;
```
Comment (koush) [inline plugins/webrtc/src/main.ts:796]: This also looks copy pasted from other code, should consolidate.

File: plugins/webrtc/src/main.ts [hunk 2]
```diff
@@ -727,15 +715,17 @@ export async function fork() {
     }
 }
 
-class WebRTCBridge extends ScryptedDeviceBase implements BufferConverter {
+class WebRTCBridge extends ScryptedDeviceBase implements MediaConverter {
```
Comment (koush) [inline plugins/webrtc/src/main.ts:718]: it looks like this was consolidated into webrtc bridge, which is itself just a buffer converter. the plugin is a buffer converter as well. let's consolidate all 3.

Comment (koush) [reply plugins/webrtc/src/main.ts:718]: I actually added media converter for webrtc plugin and rebroadcast plugin to better handle multiple conversions. just had to wait a reasonable amount of time for servers to update.

**Comment by koush** [comment]
awesome glad to finally clean this up

**Pull request merged**